lib/publiccloud/utils.pm

```diff
@@ -172,4 +184,8 @@ sub is_gce() {
     return is_public_cloud && check_var('PUBLIC_CLOUD_PROVIDER', 'GCE');
 }
 
+sub is_chost() {
```

Comment:
I would recommend to use the full name instead of the abbreviation here to reduce the number of potentially unknown abbreviations

```suggestion
sub is_container_host() {
```

Pro: Learns new members and other people what `chost` means  
Con: Longer to write

WDYT?